Description: Upstream changes introduced in version 0.14-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 protobuf-c (0.14-1) unstable; urgency=low
 .
   * New upstream release.
 .
 The person named in the Author field signed this changelog entry.
Author: Robert S. Edmonds <edmonds@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- protobuf-c-0.14.orig/src/test/Makefile.am
+++ protobuf-c-0.14/src/test/Makefile.am
@@ -1,4 +1,4 @@
-check_PROGRAMS = test-generated-code test-generated-code2 test-rpc
+check_PROGRAMS = test-generated-code test-generated-code2
 noinst_PROGRAMS = cxx-generate-packed-data example-server example-client
 INCLUDES = -I$(srcdir)/..
 test_generated_code_SOURCES = \
@@ -11,11 +11,6 @@ test-generated-code2.c \
 generated-code/test-full.pb-c.c
 test_generated_code2_LDADD = \
 ../libprotobuf-c.la
-test_rpc_SOURCES = \
-test-rpc.c \
-generated-code/test.pb-c.c
-test_rpc_LDADD = \
-../libprotobuf-c.la
 cxx_generate_packed_data_SOURCES = \
 cxx-generate-packed-data.cc generated-code/test-full.pb.cc
 cxx-generate-packed-data.$(OBJEXT): generated-code/test-full.pb.h
@@ -45,5 +40,5 @@ BUILT_SOURCES = generated-code/test.pb-c
                 generated-code/test-full.pb.cc generated-code/test-full.pb.h \
                 generated-code/test-full-cxx-output.inc
 DISTCLEANFILES = $(BUILT_SOURCES)
-TESTS = test-generated-code test-generated-code2 test-rpc
+TESTS = test-generated-code test-generated-code2
 EXTRA_DIST = test.proto test-full.proto common-test-arrays.h
